config/locales/client.en.yml

```diff
@@ -2203,6 +2203,9 @@ en:
       optional: optional
       errors_summary_title: "This form contains errors:"
       dirty_form: "You didn't submit your changes! Are you sure you want to leave?"
+      select:
+        select_placeholder: "Select…"
+        unselect_placeholder: "Unselect…"
```

Comment:
Small nit, but I wonder if this would read more naturally as "None"?

Reply:
I agree, "None" feels more natural

Reply:
Sure :+1: